client/components/NavBar.jsx

```diff
@@ -38,7 +40,7 @@ export default function NavBar() {
 
   useEffect(() => {
     setLoggedIn(localStorage.getItem("token"));
-  });
+  }, [setLoggedIn]);
```

Comment:
When the dependency is setLoggedIn, it doesnt always update immediately compared to when it checks for the login status on render.

Reply:
You can see this when you press logout. Itll take you to the homepage, but the text will remain as logout instead of changing to login now.

Reply:
I see. I got a warning when I ran lint for not including setLoggedIn in the dependency array, so do we just have to ignore that?